src/views/components/Image.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import 'views/assets/stylesheets/Image.css';
 
-export const Image = ({image}) => {
+export const Image = ({image, selectedImageUrl, handleClick}) => {
+  const selectedClass =
+    selectedImageUrl && selectedImageUrl === image.url ? 'selected' : '';
```

Comment:
Is this ok?

Reply:
I'm changing the results of the ternary operator to be 'Image selected' and 'Image' for readability.